src/_data/sidenav.yml

```
@@ -115,8 +115,9 @@
           permalink: /ui/advanced/gestures
         - title: Shaders
           permalink: /ui/advanced/shaders
+    - title: Scrolling
```

Looks like this is causing the build to fail, since `children` isn't specified. 

Is this what you intended:

```suggestion
    - title: Scrolling
      permalink: /ui/scrolling
      children:
        - title: Introduction
          permalink: /ui/scrolling
          match-page-url-exactly: true
```

Also consider moving this section above "Advanced UI".